code/backend/batch/utilities/search/IntegratedVectorizationSearchHandler.py

```diff
@@ -1,4 +1,4 @@
-from backend.batch.utilities.search.SearchHandlerBase import SearchHandlerBase
```

Comment:
How come this was building fine but only failing after deployment?

Is there any way this type of problem can be prevented in the future?

Reply:
This needs to be looked into. Will raise an issue for the same